### PR TITLE
Refactor perception

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
move skybiometry to be always launched

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Merge together with: tue-robotics/tue_robocup#493 and tue-robotics/robot_launch_files#20

**Comment by LoyVanBeek** [comment]
Looks fine Is there any reason we didn't always include it?

**Comment by MatthijsBurgh** [comment]
Not always needed, but now it is build in the robot_skills.

File: launch/state_machines/challenge_spr.launch [hunk 1]
```diff
@@ -23,9 +23,11 @@
         </group>
 
         <!--Launch SkyBiometry rosrun skybiometry_ros face_properties_node-->
+<!--
         <node pkg="skybiometry_ros" type="face_properties_node" name="skybiometry_face_properties">
                 <param name="api_timeout" value="60" />
         </node>
+-->
```
Comment (Rayman) [inline launch/state_machines/challenge_spr.launch]: If you are removing code, please remove it

**Review by Rayman — DISMISSED** [review]

File: launch/state_machines/challenge_spr.launch (hunk 1 above)
Comment (MatthijsBurgh) [reply launch/state_machines/challenge_spr.launch]: Oh that was the plan

**Review by samaleksandrov — APPROVED** [review]

**Pull request merged**